### PR TITLE
l10n: translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [ubuntu-desktop-installer/ubuntu-bootstrap](https://hosted.weblate.org/projects/ubuntu-desktop-installer/ubuntu-bootstrap/).

It also includes following components:

* [ubuntu-desktop-installer/ubuntu-provision](https://hosted.weblate.org/projects/ubuntu-desktop-installer/ubuntu-provision/)

* [ubuntu-desktop-installer/ubuntu-init](https://hosted.weblate.org/projects/ubuntu-desktop-installer/ubuntu-init/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb [hunk 1]
```diff
@@ -14,63 +14,63 @@
...
+  "telemetryLabelOff": "Ні, не ділиться даними системи",
   "@telemetryLabelOff": {},
-  "privacyPageTitle": "Сервіси місцезнаходження хитрих єврейських розробників устрою",
+  "privacyPageTitle": "Сервіси визначення місцезнаходження",
   "@privacyPageTitle": {},
-  "privacyLocationTitle": "Подарувати своє місценаходження хитрим євреям?",
+  "privacyLocationTitle": "Увімкнути сервіси визначення місцезнаходження?",
   "@privacyLocationTitle": {},
-  "storeHeader": "Почніть жерти лайно з більшою кількістю єврейських членів",
+  "storeHeader": "Почніть роботу з іншими програмами",
```
Comment (delgod) [inline packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb:64]:
```suggestion
  "storeHeader": "Почніть роботу з додатковими програмами",
```

File: packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb [hunk 2]
```diff
@@ -14,63 +14,63 @@
...
+  "welcomePageTitle": "Ласкаво просимо!",
   "@welcomePageTitle": {},
-  "welcomePageHeader": "{distro} для круглих педерастів",
+  "welcomePageHeader": "Ласкаво просимо в {distro}",
   "@welcomePageHeader": {},
-  "welcomeStartTitle": "Станьте круглим педерастом користуючись цим устроєм",
+  "welcomeStartTitle": "Спрощений початок",
   "@welcomeStartTitle": {},
-  "welcomeStartSubtitle": "Підтримка чистого підорства.",
+  "welcomeStartSubtitle": "Підтримка більш зручного налаштування.",
```
Comment (delgod) [inline packages/ubuntu_init/lib/src/l10n/ubuntu_init_uk.arb:47]:
```suggestion
  "welcomeStartSubtitle": "Підтримка простих налаштування.",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 3]
```diff
@@ -98,5 +98,66 @@
...
+  "@identityPasswordHide": {},
+  "activeDirectoryDomainEmpty": "Обов'язково",
+  "@activeDirectoryDomainEmpty": {},
+  "activeDirectoryErrorTitle": "Помилка при налаштовуванні з'єднання із Active Directory",
+  "@activeDirectoryErrorTitle": {},
+  "activeDirectoryDomainInvalidChars": "Неточні символи",
+  "@activeDirectoryDomainInvalidChars": {},
+  "activeDirectoryDomainNotFound": "Домен не знайдено",
+  "@activeDirectoryDomainNotFound": {},
+  "activeDirectoryDomainEndHyphen": "Закінчити тире (-)",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:161]:
```suggestion
  "activeDirectoryDomainEndHyphen": "Закінчується дефісом (-)",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 4]
```diff
@@ -98,5 +98,66 @@
...
+  "@activeDirectoryDomainTooLong": {},
+  "activeDirectoryAdminLabel": "Користувач домену",
+  "@activeDirectoryAdminLabel": {},
+  "identityPasswordHide": "Сховати",
+  "@identityPasswordHide": {},
+  "activeDirectoryDomainEmpty": "Обов'язково",
+  "@activeDirectoryDomainEmpty": {},
+  "activeDirectoryErrorTitle": "Помилка при налаштовуванні з'єднання із Active Directory",
+  "@activeDirectoryErrorTitle": {},
+  "activeDirectoryDomainInvalidChars": "Неточні символи",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:157]:
```suggestion
  "activeDirectoryDomainInvalidChars": "Заборонені символи",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 5]
```diff
@@ -98,5 +98,66 @@
...
+  "@activeDirectoryAdminEmpty": {},
+  "activeDirectoryDomainLabel": "Домен",
+  "@activeDirectoryDomainLabel": {},
+  "identityHostnameTooLong": "Ім'я комп'ютера занадто довге.",
+  "@identityHostnameTooLong": {},
+  "activeDirectoryDomainMultipleDots": "Містить кілька послідовних точок (.)",
+  "@activeDirectoryDomainMultipleDots": {},
+  "identityRealNameTooLong": "Це ім'я занадто довге.",
+  "@identityRealNameTooLong": {},
+  "activeDirectoryDomainEndDot": "Закінчити крапкою (.)",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:145]:
```suggestion
  "activeDirectoryDomainEndDot": "Закінчується крапкою (.)",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 6]
```diff
@@ -98,5 +98,66 @@
...
+  "@activeDirectoryDomainStartDot": {},
+  "activeDirectoryErrorMessage": "На жаль, зараз неможливо налаштувати Active Directory. Коли ваша система буде запущена, відвідайте <a href=\"https://help.ubuntu.com/activedirectory\">help.ubuntu.com/activedirectory</a>, щоб отримати допомогу.",
+  "@activeDirectoryErrorMessage": {},
+  "activeDirectoryAdminEmpty": "Обов'язково",
+  "@activeDirectoryAdminEmpty": {},
+  "activeDirectoryDomainLabel": "Домен",
+  "@activeDirectoryDomainLabel": {},
+  "identityHostnameTooLong": "Ім'я комп'ютера занадто довге.",
+  "@identityHostnameTooLong": {},
+  "activeDirectoryDomainMultipleDots": "Містить кілька послідовних точок (.)",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:141]:
```suggestion
  "activeDirectoryDomainMultipleDots": "Містить кілька послідовних крапок (.)",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 7]
```diff
@@ -98,5 +98,66 @@
...
+  "@activeDirectoryTitle": {},
+  "activeDirectoryAdminInvalidChars": "Неточні символи",
+  "@activeDirectoryAdminInvalidChars": {},
+  "activeDirectoryTestConnection": "Тестування підключення домену",
+  "@activeDirectoryTestConnection": {},
+  "identityPasswordShow": "Показати",
+  "@identityPasswordShow": {},
+  "identityActiveDirectoryInfo": "На наступному кроці ви введете назву домену та інші подробиці.",
+  "@identityActiveDirectoryInfo": {},
+  "activeDirectoryPasswordEmpty": "Обов'язково",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:125]:
```suggestion
  "activeDirectoryPasswordEmpty": "Обов'язково заповнити",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 8]
```diff
@@ -98,5 +98,66 @@
...
+  "@activeDirectoryPasswordEmpty": {},
+  "activeDirectoryDomainStartHyphen": "Почати з тире (-)",
+  "@activeDirectoryDomainStartHyphen": {},
+  "activeDirectoryPasswordLabel": "Пароль",
+  "@activeDirectoryPasswordLabel": {},
+  "activeDirectoryDomainStartDot": "Почати з крапки (.)",
+  "@activeDirectoryDomainStartDot": {},
+  "activeDirectoryErrorMessage": "На жаль, зараз неможливо налаштувати Active Directory. Коли ваша система буде запущена, відвідайте <a href=\"https://help.ubuntu.com/activedirectory\">help.ubuntu.com/activedirectory</a>, щоб отримати допомогу.",
+  "@activeDirectoryErrorMessage": {},
+  "activeDirectoryAdminEmpty": "Обов'язково",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:135]:
```suggestion
  "activeDirectoryAdminEmpty": "Обов'язково заповнити",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 9]
```diff
@@ -98,5 +98,66 @@
...
+      "DISTRO": {}
+    }
+  },
+  "identityActiveDirectoryOption": "Скористатися Active Directory",
+  "@identityActiveDirectoryOption": {},
+  "activeDirectoryTitle": "Налаштування Active Directory",
+  "@activeDirectoryTitle": {},
+  "activeDirectoryAdminInvalidChars": "Неточні символи",
+  "@activeDirectoryAdminInvalidChars": {},
+  "activeDirectoryTestConnection": "Тестування підключення домену",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:119]:
```suggestion
  "activeDirectoryTestConnection": "Тестування з'єднання з доменом",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 10]
```diff
@@ -98,5 +98,66 @@
...
+    "type": "text",
+    "placeholders": {
+      "DISTRO": {}
+    }
+  },
+  "identityActiveDirectoryOption": "Скористатися Active Directory",
+  "@identityActiveDirectoryOption": {},
+  "activeDirectoryTitle": "Налаштування Active Directory",
+  "@activeDirectoryTitle": {},
+  "activeDirectoryAdminInvalidChars": "Неточні символи",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:117]:
```suggestion
  "activeDirectoryAdminInvalidChars": "Заборонені символи",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb [hunk 11]
```diff
@@ -98,5 +98,66 @@
   "networkHiddenWifiNameRequired": "Необхідно вказати назву мережі",
   "@networkHiddenWifiNameRequired": {},
   "networkNoneOption": "Я не хочу зараз підключатися до Інтернету",
-  "@networkNoneOption": {}
+  "@networkNoneOption": {},
+  "keyboardDetectButton": "Виявити",
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_uk.arb:102]:
```suggestion
  "keyboardDetectButton": "Визначити",
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart [hunk 12]
```diff
@@ -124,73 +124,73 @@ class UbuntuProvisionLocalizationsUk extends UbuntuProvisionLocalizations {
...
-  String get activeDirectoryDomainLabel => 'Domain';
+  String get activeDirectoryDomainLabel => 'Домен';
 
   @override
-  String get activeDirectoryDomainEmpty => 'Required';
+  String get activeDirectoryDomainEmpty => 'Обов\'язково';
 
   @override
-  String get activeDirectoryDomainTooLong => 'Too long';
+  String get activeDirectoryDomainTooLong => 'Задовгий';
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart:151]:
```suggestion
  String get activeDirectoryDomainTooLong => 'Цей домен занадто довгий';
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart [hunk 13]
```diff
@@ -124,73 +124,73 @@ class UbuntuProvisionLocalizationsUk extends UbuntuProvisionLocalizations {
...
 
   @override
-  String get activeDirectoryDomainInvalidChars => 'Invalid characters';
+  String get activeDirectoryDomainInvalidChars => 'Заборонені символи';
 
   @override
-  String get activeDirectoryDomainStartDot => 'Starts with a dot (.)';
+  String get activeDirectoryDomainStartDot => 'Почати з крапки (.)';
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart:157]:
```suggestion
  String get activeDirectoryDomainStartDot => 'Починається з крапки (.)';
```

File: packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart [hunk 14]
```diff
@@ -124,73 +124,73 @@ class UbuntuProvisionLocalizationsUk extends UbuntuProvisionLocalizations {
...
 
   @override
-  String get activeDirectoryDomainEndDot => 'Ends with a dot (.)';
+  String get activeDirectoryDomainEndDot => 'Закінчується крапкою (.)';
 
   @override
-  String get activeDirectoryDomainStartHyphen => 'Starts with a hyphen (-)';
+  String get activeDirectoryDomainStartHyphen => 'Почати з тире (-)';
```
Comment (delgod) [inline packages/ubuntu_provision/lib/src/l10n/ubuntu_provision_localizations_uk.dart:163]:
```suggestion
  String get activeDirectoryDomainStartHyphen => 'Починається з тире (-)';
```

**Review by delgod — APPROVED** [review]

**Review by taurus-forever — APPROVED** [review]
UK files are OK.

**Review by delgod — APPROVED** [review]
Ukraine translation is ok

**Review by d-loose — APPROVED** [review]

**Comment by d-loose** [comment]
@seb128, @tim-hm
I've merged @delgod's suggestions here, resolved the conflicts and regenerated the l10n strings. I'd propose merging this and afterwards taking a look at #172 for any remaining translations.

**Pull request merged**